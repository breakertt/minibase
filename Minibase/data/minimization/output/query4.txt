Q(x, y) :- T(x, a), R(y, x, a)