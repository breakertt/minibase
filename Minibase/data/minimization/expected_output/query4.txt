Q(x, y) :- R(y, x, a), T(x, a)